_includes/contact_form.html

```diff
@@ -530,16 +530,14 @@ <h3>{{ site.data.[page.lang].settings.contact_page.support_request_form.title }}
       ' }}
     </div>
 
-    <div class="usa-alert usa-alert--warning display-none margin-bottom-4" id="pii-warning">
-      <div class="usa-alert__body">
-        <p
-          class="usa-alert__text"
-          id="pii-warning-message"
-          role="alert"
-          data-error="{{ site.data.[page.lang].settings.contact_page.support_request_form.errors.pii_found }}"
-        ></p>
-      </div>
-    </div>
+    {% assign pii_alert_content = site.data.[page.lang].settings.contact_page.support_request_form.errors.pii_found %}
+    {% include alert.html
+       content=pii_alert_content
```

Comment:
we probably don't need the `assign` here right? we could just inline it?
```suggestion
    {% include alert.html
       content=site.data.[page.lang].settings.contact_page.support_request_form.errors.pii_found
```

Reply:
Nope, tried that, it's a syntax error apparently 🤷